### DISCUSSION TITLE
Names groups and staves individually

### DISCUSSION THREAD

**Discussion by chrisfredd** [musescore/MuseScore#25923] [Ideas]
Would it be possible to independently give a name to groups of instruments as well as individual staves in orchestras scores? ie. have Flutes sub-bracketed and each staves numbered (see below)

<img width="306" alt="Instrument Groups" src="https://github.com/user-attachments/assets/fc0d9164-cdaf-4640-adf6-6365d026c8b1" />

**Comment by sammik** [comment]
Group name and brackets are possible (click Instruments tab, and than Add staff).

https://github.com/user-attachments/assets/7dae0363-3809-48ab-b7eb-e3ed6e852766

But staff numbers You probably need to fake. Add frame to get space between instrument name and staff, and then Staff text with number to each staff and place it by hand to the begining of staff.

https://github.com/user-attachments/assets/9bd143b9-3bbf-4ada-a3e9-9695ec8d8071

**Reply by chrisfredd** [reply]
Hi there!

Thanks for the reply! I was initially able to quickly add a new stave as you did but was hoping not to have to "fake it". I was rather thinking about this as a possible new feature?

**Reply by ghost** [reply]
This is absolutely coming in MuseScore 5.0 as part of a proper overhaul of how instruments work and are labelled; see [this forum post](https://musescore.org/en/4.5-and-beyond).
